examples/camera_server/camera_server.cpp

```diff
@@ -0,0 +1,102 @@
+#include <future>
+
+#include <iostream>
+#include <map>
+#include <thread>
+#include <mavsdk/mavsdk.h>
+#include <mavsdk/plugins/camera/camera.h>
+#include <mavsdk/plugins/camera_server/camera_server.h>
+
+using namespace mavsdk;
+
+using std::chrono::duration_cast;
+using std::chrono::seconds;
+using std::chrono::milliseconds;
+using std::chrono::system_clock;
+using std::this_thread::sleep_for;
+
+int main(int argc, char** argv)
+{
+    mavsdk::Mavsdk mavsdk;
+    mavsdk::Mavsdk::Configuration configuration(mavsdk::Mavsdk::Configuration::UsageType::Camera);
+    mavsdk.set_configuration(configuration);
+
+    // 14030 is the default camera port for PX4 SITL
+    // FIXME: the camera should probably initiate the connection and not just listen
+    auto result = mavsdk.add_any_connection("udp://:14030");
```

Comment:
Doesn't something like this work?

```
auto result = mavsdk.add_any_connection("udp://127.0.0.1:14550");
```

It should send heartbeats to `udp://127.0.0.1:14550`

Reply:
14030 is the correct choice, at least from PX4 point of view as it's the default payload port: [udp_onboard_payload_port_remote=$((14030+px4_instance))](https://github.com/PX4/PX4-Autopilot/blob/master/ROMFS/px4fmu_common/init.d-posix/px4-rc.mavlink#L8)

Reply:
Right, but shouldn't the camera_server send heartbeats to 14030, instead of listening on 14030? That would be `udp://127.0.0.1:14030` instead of `udp://:14030`

Reply:
No, it's the "remote port"

Reply:
So you're saying that a camera_server component should not send heartbeats and instead wait for heartbeats on 14030? I don't really mind (both ways are possible), but I thought the typical mavlink way is that the camera broadcasts its heartbeats and clients discover it.

Reply:
I think listening to a port is superior in this case. You can't expect the client to be on 127.0.0.1 as well. Imagine a drone hosting the camera_server and QGC connects to it, trying to read the camera_information from it. QGC will most likely not be on 127.0.0.1.

Reply:
Ok. I don't mind so much, it's just an example :slightly_smiling_face: